dist/redhat/scylla-cqlsh.spec

```diff
@@ -0,0 +1,41 @@
+Name:           %{product}-cqlsh
+Version:        %{version}
+Release:        %{release}%{?dist}
+Summary:        Scylla cqlsh tool
+Group:          Applications/Databases
+
+License:        Apache
+URL:            http://www.scylladb.com/
+Source0:        %{reloc_pkg}
+BuildArch:      noarch
+Requires:       python3
+AutoReqProv:    no
+Conflicts:      cassandra
+
+%description
+
+%global __brp_python_bytecompile %{nil}
+%global __brp_mangle_shebangs %{nil}
+
+%prep
+%setup -q -n scylla-cqlsh
+
+
+%build
+
+%install
+rm -rf $RPM_BUILD_ROOT
+./install.sh --root "$RPM_BUILD_ROOT"
+
+%files
+%{_bindir}/cqlsh
+%{_bindir}/cqlsh.py
+/opt/scylladb/share/cassandra/bin/cqlsh
+/opt/scylladb/share/cassandra/bin/cqlsh.py
+/opt/scylladb/share/cassandra/pylib/*
+/opt/scylladb/share/cassandra/lib/*
+
+
+%changelog
+* Fri Aug  7 2015 Takuya ASADA Takuya ASADA <syuu@cloudius-systems.com>
```

Comment:
Bad copy

Reply:
I think we never do edit the changelog on our deb/rpm, can they completed be removed ?

Reply:
I don't know, rpm-build may complain, or not.

Reply:
It only does a warning if it's empty